src/api/wix_client.py

```
@@ -1,11 +1,11 @@
-# src/api/wix_client.py - ENHANCED VERSION WITH NEW ORDER ENDPOINTS
+# src/api/wix_client.py - ENHANCED VERSION WITH RENDER TYPE SUPPORT
 import aiohttp
 import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)